SocialCareCaseViewerApi.Tests/V1/Gateways/DatabaseGateway.Tests.cs

```diff
@@ -420,11 +420,37 @@ public void CreatingAnAllocationShouldInsertIntoTheDatabase()
             var query = DatabaseContext.Allocations;
             var insertedRecord = query.First(x => x.Id == response.AllocationId);
 
+            insertedRecord.RagRating.Should().Be(request.RagRating);
+            insertedRecord.Summary.Should().Be(request.Summary);
+            insertedRecord.CarePackage.Should().Be(request.CarePackage);
             insertedRecord.PersonId.Should().Be(request.MosaicId);
             insertedRecord.WorkerId.Should().Be(worker.Id);
             insertedRecord.CreatedBy.Should().Be(createdByWorker.Email);
         }
 
+        [Test]
```

Comment:
Should tests for other scenarios be covered in this PR as well? For example  creating an allocation without a team and without a team and worker